.gitignore

```diff
@@ -1,4 +1,10 @@
 minecraft_recipes.db
-modules/__pycache__/*
-/__pycache__/*
-tests/__pycache__/*
\ No newline at end of file
+mc_calculator/__pycache__
+/__pycache__
+tests/__pycache__
+/test_env/*
+/test_env
+/dist
+/mc_calculator.egg-info
+/build
+/.pytest_cache
```

Comment:
Probably optional, but you want to get into the habit if possible of having a newline at the end of your files generally. [Here's a good article about topic](https://medium.com/@alexey.inkin/how-to-force-newline-at-end-of-files-and-why-you-should-do-it-fdf76d1d090e) including references to the posix standard.

Could also punt this kind of work to when you set up CICD for linting, but that will only cover python source files. The .gitignore you'll want to do manually.

Reply:
I'll definitely do that.